specification/source/cdl.rst

```diff
@@ -1688,118 +1688,97 @@ Therefore, tools that process CDL can infer the following information:
+The buildings industry uses different metadata schemes such as
+`Brick <https://brickschema.org/>`_, `Project Haystack <https://project-haystack.org/>`_ and `ASHRAE 223p <https://www.ashrae.org/about/news/2018/ashrae-s-bacnet-committee-project-haystack-and-brick-schema-collaborating-to-provide-unified-data-semantic-modeling-solution>`_. CDL allows, but does not require, use of the Brick or Haystack or any other tagging scheme.
 
-CDL allows to add tags to declarations that instantiate
-
-* elementary blocks (:numref:`sec_ele_blo`), and
-* composite blocks (:numref:`sec_com_blo`).
-
-[We currently do not see a use case that would require
-adding a tag to a ``parameter`` declaration.]
+* Semantic information will be included within the ``annotation`` keyword, using the ``__Buildings`` or ``__cdl`` annotation. ``__cdl`` will be used when the semantic information is regarding a control sequence (inputs, outputs etc.) and ``__Buildings`` will be used for every other instance that are not a part of the control sequence (equipment, zones etc.). The following instances defined in CDL can have annotations containing semantic information:
+ * Inputs
+ * Outputs
+ * Parameters
+ * Composite blocks (:numref:`sec_com_blo`)
+ * Packages
+ 
+* All semantic information will be included under the ``semantic`` section within the ``__Building`` and/or ``__cdl`` annotations. 
 
+.. code-block:: modelica
 
-To implement such tags, CDL blocks can contain
-vendor annotations with the following syntax:
+   annotation (__cdl(semantic(<semantic information>)))
+   annotation (__Buildings(semantic(<semantic information>)))
+   
+* The top-level block declaration shall house the ``metadataLanguageDefinition`` within the  ``semantic`` section. ``metadataLanguageDefinition`` will be used to define the different types of tags that are used throughout the CDL block and relevant information about the same. It will follow the following syntax:
 
 .. code-block:: modelica
 
-   annotation :
-     annotation "(" [annotations ","]
-        __cdl "(" [ __cdl_annotation ] ")" ["," annotations] ")"
+   annotation (__cdl(semantic(metadataLanguageDefinition="<metadataLanguage> [version] [format]" "<description or URL>")))
+   annotation (__Buildings(semantic(metadataLanguageDefinition="<metadataLanguage> [version] [format]" "<description or URL>")))
 
-where ``__cdl__annotation`` is the annotation for CDL.
-
-For Brick, the ``__cdl_annotation`` is
+* ``metadataLanguageDefinition`` could be semantic data standards such as  Brick, Project-Haystack, ASHRAE 223p etc. or it could just be a natural language description. The optional ``version`` represents the version of the metadataLangauge used in a particular sequence and the optional ``format``  represents the format this will be expressed in.  The format will be expressed using  `MIME types <https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types>`_. Examples:
```

Comment:
When stating an example, put it within brackes such as
```
[
...
]
```
See item 2 in http://obc.lbl.gov/specification/conventions.html